### PR TITLE
CHANGELOG: update for 0.22.0-rc.0

### PR DESCRIPTION
**Pull request by GiedriusS**
Update CHANGELOG & tutorials for the 0.22.0-rc.0. I haven't spotted any
metrics changes or any incompatibilities.

Signed-off-by: Giedrius Statkevičius <giedrius.statkevicius@vinted.com>

### CODE REVIEW

**Review by ianbillett — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,17 +22,23 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
...
 - [#4388](https://github.com/thanos-io/thanos/pull/4388) Receive: fix bug in forwarding remote-write requests within the hashring via gRPC when TLS is enabled on the HTTP server but not on the gRPC server.
+- [#4340](https://github.com/thanos-io/thanos/pull/4340) UI: now displays the duration and all annotations of an alert in the alerts page
+- [#4348](https://github.com/thanos-io/thanos/pull/4348) Fixed parsing of the port in the log middleware
+- [#4417](https://github.com/thanos-io/thanos/pull/4417) UI: fixed the night mode in Bucket UI
 
 ### Changed
 
 - [#4354](https://github.com/thanos-io/thanos/pull/4354) Receive: use the S2 library for decoding Snappy data; saves about 5-7% of CPU time in the Receive component when handling incoming remote write requests
-- [#4369](https://github.com/thanos-io/thanos/pull/4354) Build: do not upgrade apline version
+- [#4369](https://github.com/thanos-io/thanos/pull/4354) Build: do not upgrade Alpine's version
```
Comment (bwplotka) [inline CHANGELOG.md:40]:
```suggestion
- [#4369](https://github.com/thanos-io/thanos/pull/4354) Build: Pin upgrade Alpine's version.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,17 +22,23 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
...
+- [#3678](https://github.com/thanos-io/thanos/pull/3678) UI: add support for new duration format in graph range input. Now it is possible to use composite durations such as `5m30s`.
 
 ### Fixed
 
 - [#4384](https://github.com/thanos-io/thanos/pull/4384) Fix the experimental PromQL editor when used on multiple line.
 - [#4342](https://github.com/thanos-io/thanos/pull/4342) ThanosSidecarUnhealthy doesn't fire if the sidecar is never healthy
 - [#4388](https://github.com/thanos-io/thanos/pull/4388) Receive: fix bug in forwarding remote-write requests within the hashring via gRPC when TLS is enabled on the HTTP server but not on the gRPC server.
+- [#4340](https://github.com/thanos-io/thanos/pull/4340) UI: now displays the duration and all annotations of an alert in the alerts page
+- [#4348](https://github.com/thanos-io/thanos/pull/4348) Fixed parsing of the port in the log middleware
+- [#4417](https://github.com/thanos-io/thanos/pull/4417) UI: fixed the night mode in Bucket UI
```
Comment (bwplotka) [inline CHANGELOG.md:35]:
```suggestion
- [#4417](https://github.com/thanos-io/thanos/pull/4417) UI: fixed the night mode in Bucket UI.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -22,17 +22,23 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
...
+- [#3678](https://github.com/thanos-io/thanos/pull/3678) UI: add support for new duration format in graph range input. Now it is possible to use composite durations such as `5m30s`.
 
 ### Fixed
 
 - [#4384](https://github.com/thanos-io/thanos/pull/4384) Fix the experimental PromQL editor when used on multiple line.
 - [#4342](https://github.com/thanos-io/thanos/pull/4342) ThanosSidecarUnhealthy doesn't fire if the sidecar is never healthy
 - [#4388](https://github.com/thanos-io/thanos/pull/4388) Receive: fix bug in forwarding remote-write requests within the hashring via gRPC when TLS is enabled on the HTTP server but not on the gRPC server.
+- [#4340](https://github.com/thanos-io/thanos/pull/4340) UI: now displays the duration and all annotations of an alert in the alerts page
+- [#4348](https://github.com/thanos-io/thanos/pull/4348) Fixed parsing of the port in the log middleware
```
Comment (bwplotka) [inline CHANGELOG.md:34]:
```suggestion
- [#4348](https://github.com/thanos-io/thanos/pull/4348) Fixed parsing of the port in the log middleware.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -22,17 +22,23 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
...
+- [#4231](https://github.com/thanos-io/thanos/pull/4231) Receive: implemented the new dual-mode according to this [doc](https://github.com/thanos-io/thanos/blob/release-0.22/docs/proposals-accepted/202012-receive-split.md). This means that it is now possible to not run a TSDB on a Thanos Receive instance if it only forwards write requests.
+- [#3678](https://github.com/thanos-io/thanos/pull/3678) UI: add support for new duration format in graph range input. Now it is possible to use composite durations such as `5m30s`.
 
 ### Fixed
 
 - [#4384](https://github.com/thanos-io/thanos/pull/4384) Fix the experimental PromQL editor when used on multiple line.
 - [#4342](https://github.com/thanos-io/thanos/pull/4342) ThanosSidecarUnhealthy doesn't fire if the sidecar is never healthy
 - [#4388](https://github.com/thanos-io/thanos/pull/4388) Receive: fix bug in forwarding remote-write requests within the hashring via gRPC when TLS is enabled on the HTTP server but not on the gRPC server.
+- [#4340](https://github.com/thanos-io/thanos/pull/4340) UI: now displays the duration and all annotations of an alert in the alerts page
```
Comment (bwplotka) [inline CHANGELOG.md:33]:
```suggestion
- [#4340](https://github.com/thanos-io/thanos/pull/4340) UI: now displays the duration and all annotations of an alert in the alerts page.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -22,17 +22,23 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#3970](https://github.com/thanos-io/thanos/pull/3970) Azure: Adds more configuration options for Azure blob storage. This allows for pipeline and reader specific configuration. Implements HTTP transport configuration options. These options allows for more fine-grained control on timeouts and retries. Implements MSI authentication as second method of authentication via a service principal token.
 - [#4406](https://github.com/thanos-io/thanos/pull/4406) Tools: Add retention command for applying retention policy on the bucket.
 - [#4430](https://github.com/thanos-io/thanos/pull/4430) Compact: Add flag `downsample.concurrency` to specify the concurrency of downsampling blocks.
+- [#4231](https://github.com/thanos-io/thanos/pull/4231) Receive: implemented the new dual-mode according to this [doc](https://github.com/thanos-io/thanos/blob/release-0.22/docs/proposals-accepted/202012-receive-split.md). This means that it is now possible to not run a TSDB on a Thanos Receive instance if it only forwards write requests.
```
Comment (bwplotka) [inline CHANGELOG.md:25]:
```suggestion
- [#4231](https://github.com/thanos-io/thanos/pull/4231) Receive: Implemented the [receive dual-mode](https://github.com/thanos-io/thanos/blob/release-0.22/docs/proposals-accepted/202012-receive-split.md). This means that it is now you can run receive as ingester (TSDB only, no hashring awareness), as router (no TSDB only forwarding mode) and old mode (router + ingester) in one service. Consider splitting functionality if you want to scale your hashring into multiple rings or you want horizontally autoscale your ingester more often.
```

**Review by bwplotka — APPROVED** [review]
Thank you! Some nits. Otherwise LGTM (:

**Pull request merged**